src/yearpicker/YearPickerTop.js

```diff
@@ -25,6 +27,8 @@ export default class YearPickerTop extends React.Component {
     const endDecadeYear = startDecadeYear + 9;
 
     return {
+      prevIconClassName: this.props.prevIconClassName,
```

Comment:
`getLocals` receives `props` as argument. You should get these two values from there:
```js
getLocals({ prevIconClassName, nextIconClassName }) {
```